backend/app/main/views.py

```diff
@@ -88,6 +88,25 @@ def get_all_clients():
         if name is not None:
             m = Client(name=name, email=email, notes=notes, attachments=attachments)
             list_of_clients.append(m.serialize())
+            
+    # Pagination setting that will continue to send requests until all of the records have been retrieved
+    while 'offset' in response_json: 
+        offset = response_json["offset"]
+        response = requests.get(
+        "https://api.airtable.com/v0/appw4RRMDig1g2PFI/Clients?offset={}".format(offset),
+        headers={"Authorization": str(os.environ.get("API_KEY"))},
+        )
+        response_json = response.json()
+        for r in response_json["records"]:
+            name = r["fields"].get("Name")
+            notes = r["fields"].get("Notes")
+            email = r["fields"].get("Client Email")
+            attachments = r["fields"].get("Attachments")
+            if name is not None:
+                m = Client(name=name, email=email, notes=notes, attachments=attachments)
+                list_of_clients.append(m.serialize())
+        if 'offset' in response_json:
```

Comment:
I think this is redundant since the same thing is done at the start of the loop

Reply:
Yes, good catch. Will change this and merge!